pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -82,6 +82,7 @@ func (c *client) CreateService(ctx context.Context, service types.Service) (*typ
 		DeploymentController:          service.DeploymentController,
 		DeploymentConfiguration:       service.DeploymentConfiguration,
 		EnableECSManagedTags:          service.EnableECSManagedTags,
+		EnableExecuteCommand:          service.EnableExecuteCommand,
```

Comment:
Nice, could you also update the UpdateService function as well, I guess someone also needs this field either.

ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/platformprovider/ecs/client.go#L110

Reply:
Thanks. I added at e15a682.